pyfstat/make_sfts.py

```diff
@@ -394,45 +394,49 @@ def make_cff(self, verbose=False):
         else:
             logging.info("Writing config file {:s}...".format(self.config_file_name))
 
-        if self.check_if_cff_file_needs_rewritting(content):
+        if self.check_if_cff_file_needs_rewriting(content):
             config_file = open(self.config_file_name, "w+")
             config_file.write(content)
             config_file.close()
 
     def check_cached_data_okay_to_use(self, cl_mfd):
-        """ Check if cached data exists and, if it does, if it can be used """
+        """ Check if matching data already exists in SFT files and can be used. """
```

Comment:
This line suggest a stronger condition than what it actually does: We don NOT check that the SFTs contain the signal present in the `.cff`, rather we

1) Check that SFT files are actually present.
2) Asses that SFT files were created after the `.cff` file.
3) Check that a file with the same name as said `.cff` file was used in the SFT creation.

This method does **not** check for the `.cff` contents, nor that should be suggested, as one has to manually trigger another method to do so. 

I could accept that `...if maching data...` actually refers to such condition, but looks unclear to me.

```suggestion
        """ Check if a signal was injected into the SFT files and these can be re-used. """
```

Reply:
Good point, though the commandline consistency check is actually quite powerful. Also your suggestion about "a signal was injected" is also too strong by the same logic. Let's go with an extended docstring:
```suggestion
        """ Check if SFT files already exist that can be re-used.
        
             This does not check the actual data contents of the SFTs,
             but only the following criteria:
              * filename
              * if an injection is requested, that the .cff file is older than the SFTs
                (but its contents should have been checked separately)
              * that the commandline stored in the (first) SFT header matches
        """
```